src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -95,232 +32,19 @@ const NavBar: React.FC<NavBarPropsType> = ({
       : undefined
   );
 
-  const {
-    location: { pathname },
-  } = useHistory();
-  const isOnPlaya = pathname.toLowerCase() === venueInsideUrl(PLAYA_VENUE_ID);
-
-  const now = firebase.firestore.Timestamp.fromDate(new Date());
-  const futureUpcoming =
-    venue?.events?.filter((e) => e.ts_utc.valueOf() > now.valueOf()) ?? []; //@debt typing does this exist?
-
-  const hasUpcomingEvents = futureUpcoming && futureUpcoming.length > 0;
-
-  const hasRadioStations = radioStations && radioStations.length;
-  const isSoundCloud =
-    !!hasRadioStations && RegExp("soundcloud").test(radioStations![0]);
-
-  const sound = useMemo(
-    () =>
-      radioStations && hasElements(radioStations) && !isSoundCloud
-        ? new Audio(radioStations[0])
-        : undefined,
-    [isSoundCloud, radioStations]
-  );
-
-  const [isRadioPlaying, setIsRadioPlaying] = useState(false);
-  const { volume, setVolume } = useRadio(isRadioPlaying, sound);
-
-  const radioFirstPlayStateLoaded = useRef(false);
-  const showRadioOverlay = useMemo(() => {
-    if (!radioFirstPlayStateLoaded.current) {
-      const radioFirstPlayStorageKey = "radioFirstPlay";
-      const radioFirstPlayState = localStorage.getItem(
-        radioFirstPlayStorageKey
-      );
-      if (!radioFirstPlayState) {
-        localStorage.setItem(radioFirstPlayStorageKey, JSON.stringify(true));
-        return true;
-      }
-      radioFirstPlayStateLoaded.current = true;
-    }
-    return false;
-  }, [radioFirstPlayStateLoaded]);
-
-  const [isEventScheduleVisible, setEventScheduleVisible] = useState(false);
-  const toggleEventSchedule = useCallback(() => {
-    setEventScheduleVisible(!isEventScheduleVisible);
-  }, [isEventScheduleVisible]);
-  const hideEventSchedule = useCallback(() => {
-    setEventScheduleVisible(false);
-  }, []);
-
-  const parentVenueId = venue?.parentId ?? "";
-  const backToParentVenue = useCallback(() => {
-    window.location.href = venueInsideUrl(parentVenueId);
-  }, [parentVenueId]);
```

Comment:
Where was this moved to, and/or why was it removed?

Reply:
There is a back button now in the admin sidebar which is going to be removed in my next PR because of the new Admin v3 design this is here temporary.

Reply:
You say there’s a back button in the admin sidebar, but how about in the ‘client’ side of things? Was this not used for that too?

Reply:
That's true. Adding this back.